orion_mcp.py

```diff
@@ -104,8 +107,13 @@ async def openshift_report_on(
         version=version
     )
 
+    # Log the full result for debugging
+    print(f"Orion return code: {result.returncode}")
+    print(f"Orion stdout: {result.stdout}")
+    print(f"Orion stderr: {result.stderr}")
+
     if result.returncode != 0:
```

Comment:
since we want to display regardless if there is a change found or not... could we simplify the code but removing this entire if statement?